crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -119,6 +119,10 @@ impl Diagnostic {
     pub fn get_max_history_length(&self) -> usize {
         self.max_history_length
     }
+
+    pub fn get_history(&self) -> Vec<f64> {
```

Comment:
```suggestion
    pub fn history(&self) -> Vec<f64> {
```

Elsewhere in the code base we tend to prefer this naming convention.